Cargo.toml

```diff
@@ -29,3 +33,12 @@ tokio = "^0.1"
 
 [profile.release]
 lto = true
+
+[package.metadata.release]
+sign-commit = true
+upload-doc = false
+disable-publish = true
+disable-push = true
+pre-release-commit-message = "cargo: zincati release {{version}}"
+pro-release-commit-message = "cargo: development version bump"
```

Comment:
Aside: I'm confused why this is `pro` instead of `post` if they were going for the opposite of `pre`... :smile:

Reply:
On spot. I do believe the author is a native speaker of a non-Latin-rooted language. That field has always been called that way.